INN/umbrella-citylimits
# Podcasting widget

## Changes

This pull request makes the following changes:

- creates widget for podcasts:
    - no thumbnail, just a plain-text play icon copied from Workday Minnesota work
    - choose by category
    - link in widget admin so users can get to the list of posts that are both in the chosen category and in the "Featured in Category" prominence term
    - includes styles for an optional "More Link" button at the bottom of the widget
- minor modifications to `partials/widget-content.php` to support the play icon
- updates `.btn` and `.btn-primary` styles in a direction more in line with the interactions mockup


![Screen Shot 2019-10-24 at 17 44 17 ](https://user-images.githubusercontent.com/1754187/67529136-4f209a80-f689-11e9-9ec9-67f27e366975.png)
![Screen Shot 2019-10-24 at 17 44 25 ](https://user-images.githubusercontent.com/1754187/67529137-4f209a80-f689-11e9-8efa-6b9ea93acc93.png)
![Screen Shot 2019-10-24 at 19 17 43 ](https://user-images.githubusercontent.com/1754187/67532124-1d143600-f693-11e9-98b9-ce47e1477080.png)
![Screen Shot 2019-10-24 at 19 17 48 ](https://user-images.githubusercontent.com/1754187/67532125-1d143600-f693-11e9-8486-7d0fdd393572.png)
![Screen Shot 2019-10-24 at 19 18 04 ](https://user-images.githubusercontent.com/1754187/67532126-1d143600-f693-11e9-9d11-11431698ea33.png)
![Screen Shot 2019-10-24 at 19 18 29 ](https://user-images.githubusercontent.com/1754187/67532127-1d143600-f693-11e9-9438-7ed6fba5fc27.png)


## Why



For #45 

## Testing/Questions

Features that this PR affects:

- wodgets


Questions that need to be answered before merging:

- [x] Works?
- [x] Controls make sense?
- [ ] for smaller display on mobile, would we want to align the podcast play button beside the headline?

Steps to test this PR:

1. Add a "Citylimits Podcasts Widget" to the "Homepage Bottom" widget area
2. Configure it to pull from a category
3. Verify that no posts in that category have the "Featured in Category" prominence term
4. Check that the output matches up: should be the three most-recent posts in the category
5. Add an older post in the category to that prominence term
6. Check that the output matches up: that post should appear, and then the rest of the posts in the widget should be backfilled
7. Adjust the count up and down by 3 to make sure styles look okay with n > 3 posts.
